tests/runners/airflow/tasks/test_spark_task.py

```diff
@@ -57,26 +154,27 @@ def test_get_runnable_command(self):
             [],
             trigger_rule='all_success',
             liminal_config={},
-            pipeline_config={},
+            pipeline_config={'pipeline': 'pipeline'},
             task_config=task_config
         ).get_runnable_command()
 
         self.assertEqual(actual.sort(), expected.sort())
 
+
     def test_missing_spark_arguments(self):
         task_config = {
             'application_source': 'my_app.py',
             'application_arguments': {
-                '--query': "select * from dlk_visitor_funnel_dwh_staging.fact_events where unified_Date_prt >= "
-                           "'{{yesterday_ds}}'",
+                '--query': "select * from dlk_visitor_funnel_dwh_staging.fact_events where"
+                           " unified_Date_prt >= '{{yesterday_ds}}'",
                 '--output': 'mytable'
             }
         }
 
         expected = ['spark-submit', 'my_app.py',
                     '--query',
-                    "select * from dlk_visitor_funnel_dwh_staging.fact_events where unified_Date_prt >="
-                    " '{{yesterday_ds}}'",
+                    "select * from dlk_visitor_funnel_dwh_staging.fact_events where "
```

Comment:
Can we make the table/column/database names more generic? Looks like this was lifted from a fork.